server/src/schemas/auditLogsSchema.ts

```diff
@@ -25,15 +25,15 @@ export const AuditLogsTypeDefs = gql`
 
   input LogInput {
     timeDate: DateTime!
-    user: User!
+    userID: ID!
```

Comment:
You could use the int type here instead but I think the ID type works perfectly fine aswell